sdk/build.gradle

```diff
@@ -15,7 +16,8 @@ android {
         versionCode 20170505
         versionName "0.1.0"
 
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        multiDexEnabled true
```

Comment:
I'm going to have to test this against a sample app before I can properly answer this question.